### PR TITLE
Adding analysed data and functionality to categorise in admin panel

### PR DESCRIPTION
**Pull request by stefdworschak**
# Business Analysis 🟠 Code Institute 🟠 Community Project - Pull Request 🎉🎉

## Pull Request References:
#29 

## PR contains:
- Adding data including the text fetched from the websites and analysed keywords
- Adding editable field in the "Organisation" admin panel view to edit the categories

## Breaking changes
None

## Screenshots/Gifs/Media
Screenshot from 2022-08-03 12-04-21

### CODE REVIEW

File: data/categories.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+        "name": "Software Development",
+        "keywords": []
+    },
+    {
+        "name": "E-Commerce",
+        "keywords": []
+    }
+]
```
Comment (auxfuse) [inline data/categories.json:50]: New line needed at EOF.

**Review by auxfuse — CHANGES_REQUESTED** [review]
Lots of changes in this and lots to digest. Well done Stefan, only one change needed after a lengthy review and wrapping my head around it. That pesky EOF new line 😉

File: data/categories.json (hunk 1 above)
Comment (stefdworschak) [reply data/categories.json]: Added the line.

**Comment by stefdworschak** [comment]
Thanks @auxfuse for the review and the perseverance! :sweat_smile:

**Pull request merged**